src/NET8MinimalAPISnapSnart/template.yaml

```diff
@@ -0,0 +1,82 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+
+Globals:
+  Function:
+    MemorySize: 1024
+    Architectures: ["x86_64"]
```

Comment:
Is there a reason we can't take the architecture parameter like other methods?

Reply:
Not at all :)